target/linux/ramips/dts/mt7620n_hootoo_tm05.dts

```diff
@@ -0,0 +1,129 @@
+/dts-v1/;
+
+#include "mt7620n.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "hootoo,tm05", "ralink,mt7620n-soc";
+	model = "HooToo TM05";
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		green {
+			label = "tm05:green";
+			gpios = <&gpio2 0 GPIO_ACTIVE_HIGH>;
+			default-state = "on";
+			// Enabled below line, if flashing LED desired
+			//linux,default-trigger = "timer";
+		};
+
+		blue {
+			label = "tm05:blue";
+			gpios = <&gpio3 0 GPIO_ACTIVE_LOW>;
+			default-state = "on";
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			gpios = <&gpio2 1 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+};
+
+&gpio0 {
+	status = "okay";
+};
+
+&gpio1 {
+	status = "okay";
+};
```

Comment:
Do you need gpio1?

Reply:
I can remove if you prefer. I didn't see it causing any grief, but no issue removing it!

Reply:
I won't do any harm, but I think we do not need it.

Reply:
Sure, will remove. FYI, here is what I see currently - will test after the change, make sure no issues / delta.
```
root@OpenWrt:~# cat /sys/kernel/debug/gpio
gpiochip0: GPIOs 0-23, parent: platform/10000600.gpio, 10000600.gpio:

gpiochip1: GPIOs 24-39, parent: platform/10000638.gpio, 10000638.gpio:

gpiochip2: GPIOs 40-71, parent: platform/10000660.gpio, 10000660.gpio:
 gpio-40  (                    |tm05:green          ) out hi
 gpio-41  (                    |reset               ) in  hi

gpiochip3: GPIOs 72-72, parent: platform/10000688.gpio, 10000688.gpio:
 gpio-72  (                    |tm05:blue           ) out lo
```

Reply:
OK, all good. Chip numbering slightly different, but should be good. Info below. Green not working, but I think that's something else - checking the trigger.
```
root@OpenWrt:~# cat /sys/kernel/debug/gpio
gpiochip0: GPIOs 0-23, parent: platform/10000600.gpio, 10000600.gpio:

gpiochip1: GPIOs 40-71, parent: platform/10000660.gpio, 10000660.gpio:
 gpio-40  (                    |tm05:green:wifi     ) out lo
 gpio-41  (                    |reset               ) in  hi

gpiochip2: GPIOs 72-72, parent: platform/10000688.gpio, 10000688.gpio:
 gpio-72  (                    |tm05:blue:power     ) out lo
```